README.md

```diff
@@ -0,0 +1,94 @@
+# Tribute Page
+
+> This is my first freecodecamp project on responsive design
+
+![screenshot](./img/arch.jpg)
+
+This is a webpage on the biography of Dr. Norman Borlaug, his lifetime and achievements.
+
+## Built With
+
+- HTML,
+- CSS,
```

Comment:
We can add what type/part of HTML and CSS used here too, semantic HTML? CSS typography? etc.